content/blog/2017/2017-01-12-scm-api-2.0-release.adoc

```diff
@@ -0,0 +1,270 @@
+---
+:layout: post
+:title: SCM API turns 2.0 and what that means for you
+:tags:
+- development
+- plugins
+:author: stephenc
+---
+
+We are announcing the
+link:https://plugins.jenkins.io/scm-api[SCM API]
+2.0.x and
+link:https://plugins.jenkins.io/branch-api[Branch API]
+2.0.x release lines.
+
+Downstream of this there are also some great improvements to a number of popular plugins including:
+
+* link:https://plugins.jenkins.io/github-branch-source[GitHub Branch Source]
+* link:https://plugins.jenkins.io/cloudbees-bitbucket-branch-source[BitBucket branch source]
+* link:https://plugins.jenkins.io/git[Git]
+* link:https://plugins.jenkins.io/mercurial[Mercurial]
+* link:https://plugins.jenkins.io/workflow-multibranch[Pipeline Multibranch]
+* link:https://plugins.jenkins.io/github-organization-folder[GitHub Organization Folders]
+
+There are some gotcha's that Jenkins administrators will need to be aware of.
+
+TIP: Always take a backup of your `JENKINS_HOME` before upgrading any plugins.
+
+We want to give you the whole story, but the take home message is this:
+
+[quote,A Jenkins Administrator]
+____
+When updating the
+link:https://plugins.jenkins.io/scm-api[SCM API]
+and/or
+link:https://plugins.jenkins.io/branch-api[Branch API]
+plugins to the 2.0.x release lines, if you have any of the
+link:https://plugins.jenkins.io/github-organization-folder[GitHub Organization Folders],
+link:https://plugins.jenkins.io/github-branch-source[GitHub Branch Source]
+and/or
+link:https://plugins.jenkins.io/cloudbees-bitbucket-branch-source[BitBucket branch source]
+plugins installed then you *must* upgrade them at the same time or Bad Things™ will happen.
+____
+
+[WARNING]
+====
+Do NOT upgrade some of these plugins but not others!
+Doing so may cause your jobs to fail to load.
+====
+
+If you don't care about the hows and whys, you can just skip down to <<tldr,this section>> but if you are curious... here we go!
+
+== The back-story
+
+Way back in September 2013 we announced the
+link:https://jenkins.io/blog/2013/09/23/literate-builds-wtf/[literate plugin],
```

Comment:
At the end of blog you capitalize "Literate".  
Do you want to do that here as well, or make them all lowercase?